src/components/modal/WelcomeModal.js

```diff
@@ -9,7 +9,11 @@ export default function WelcomeModal({ children, onClose }) {
       <Dimmed onClick={onClose} />
       <WelcomeModalWrapper>
         <div className="content">{children}</div>
-        <button className="confirm-button" onClick={onClose}>
+        <button
+          className="confirm-button"
+          data-testid="confirm-btn"
```

Comment:
```suggestion
          className="confirm-button"
          data-testid="confirm-button"
```
이 부분도 일관성 있게 작성하시면 좋을 거 같습니다 !!